libnavigation-core/src/test/java/com/mapbox/navigation/core/trip/session/BannerInstructionEventTest.kt

```diff
@@ -0,0 +1,73 @@
+package com.mapbox.navigation.core.trip.session
+
+import com.mapbox.api.directions.v5.models.BannerInstructions
+import com.mapbox.navigation.base.trip.model.RouteProgress
+import io.mockk.every
+import io.mockk.mockk
+import org.junit.Assert.assertEquals
+import org.junit.Assert.assertNull
+import org.junit.Test
+
+class BannerInstructionEventTest {
```

Comment:
This class is missing a test for:
1. Non-null banner instruction provided.
2. Null banner instruction provided.
3. `assertEquals(anyBannerInstructions, bannerInstructionEvent.latestBannerInstructions)`

Reply:
As it's always updated, isn't that covered with

https://github.com/mapbox/mapbox-navigation-android/blob/9abce409b2d7f9f564dd6e29f102ad9bc0bc5f2b/libnavigation-core/src/test/java/com/mapbox/navigation/core/trip/session/BannerInstructionEventTest.kt#L14

and 

https://github.com/mapbox/mapbox-navigation-android/blob/9abce409b2d7f9f564dd6e29f102ad9bc0bc5f2b/libnavigation-core/src/test/java/com/mapbox/navigation/core/trip/session/BannerInstructionEventTest.kt#L26

🤔

Reply:
It's not as far as I can tell.

We are not verifying that `latestBannerInstructions` is not cleared by a `null` banner instructions.